addons/mail/static/src/discuss/core/common/thread_model_patch.js

```diff
@@ -108,11 +108,9 @@ const threadPatch = {
             },
             inverse: "threadAsFirstUnread",
         });
-        this.invitedMembers = Record.many("discuss.channel.member");
-        this.last_interest_dt = Record.attr(undefined, { type: "datetime" });
-        /** @type {luxon.DateTime} */
```

Comment:
Is the type correct when running the script for d.ts? Anyway we can improve the script if we need, specific fields will help for that.

Reply:
I suppose, I copy-pasted the doc-string of existing fields, which I suppose worked with the script as the `d.ts` define some luxon fields.